tests/db/docker-compose.yml

```diff
@@ -30,6 +32,8 @@ services:
       - mysql
     build:
       context: .
+    volumes:
+      - .:/tmp/mlflow
```

Comment:
Use volumes for faster development cycle instead of copying scripts in the docker image.